frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,332 @@
+              expireInfo: '1d 2h 30m left',
+              isExpired: false
+            }
+          },
+          {
+            name: 'Brian Eno',
+            inviteLink: 'http://localhost:8000/app/join?groupId=30aFnTYz7RbqAeSLLT1cNfSSeZHN6KHUnnCssmSw6dJeB1gc7v&secret=2250',
+            isAnyoneLink: false,
+            state: {
+              description: 'Used',
+              expireInfo: '',
+              isExpired: true
+            }
+          },
+          {
+            name: 'Carl Sagan',
+            inviteLink: 'http://localhost:8000/app/join?groupId=B1gfSSeZc721XWnNFz7RbkoyHUqAeSwLT1cNHnnCssmSw6dJev&secret=1348',
+            isAnyoneLink: false,
+            state: {
+              description: 'Not used',
+              expireInfo: 'Expired',
+              isExpired: true
+            }
+          },
+          {
+            name: 'Anyone',
+            inviteLink: 'http://localhost:8000/app/join?groupId=s8LT1cNHnnCs21XWnNFz7RbNPKHUqAesmSw6dJeB1gfSSeZc7v&secret=5521',
+            isAnyoneLink: true,
+            state: {
+              description: '10/60 used',
+              expireInfo: 'Expired',
+              isExpired: true
+            }
+          }
+        ]
+      }
+    }
+  },
+  methods: {
+    unfocusSelect () {
+      this.$refs.select.blur()
+    }
+  },
+  computed: {
+    activeList () {
+      return this.ephemeral.selectbox.selectedOption === 'Active' ? this.ephemeral.dummyInviteList.filter(item => !item.state.isExpired)
+        : this.ephemeral.selectbox.selectedOption === 'All' && this.ephemeral.dummyInviteList
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../../../assets/style/_variables.scss";
+
+.c-section {
+  position: relative;
+}
+
+.c-title {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.c-invite-description {
+  margin: $spacer-sm 0;
+}
+
+.c-table {
+  width: 100%;
+  table-layout: fixed;
+  margin: $size-2 0;
+
+  tr {
+    display: grid;
+    grid-template-columns: 28.4% 36.1% 23.7% auto;
+    grid-template-areas: "name invite-link state action";
+  }
+
+  @include phone {
+    thead tr {
+      grid-template-columns: 81% auto;
+      grid-template-areas: "name action";
+    }
+
+    tbody tr {
+      grid-template-columns: 81% auto;
+      grid-template-rows: 1fr 1fr;
+      grid-template-areas:
+        "name action"
+        "state action";
+    }
+  }
+
+  th, td {
+    display: flex;
+    align-items: center;
+  }
+
+  .c-name {
+    grid-area: name;
+    padding-right: $size-2;
+
+    button {
+      display: inline-block;
+      color: $primary_0;
+    }
+  }
+
+  .c-invite-link {
+    grid-area: invite-link;
+    padding-right: $size-2;
+
+    &-wrapper {
+      display: inherit;
+      align-items: inherit;
+      width: 100%;
+    }
+
+    .c-invite-link-button {
+      padding: 0 $spacer / 3;
+      width: 1.6875rem;
+      height: 1.6875rem;
+      font-weight: normal;
+    }
+
+    @include phone {
```

Comment:
not visible on phone?

Reply:
I just realized this td needs to be moved to the end of the row on phone(with different content). Just fixed it.